circuits/cpp/barretenberg/cpp/src/barretenberg/honk/pcs/gemini/gemini.cpp

```diff
@@ -44,6 +44,11 @@
  */
 namespace proof_system::honk::pcs::gemini {
 
+/**
```

Comment:
just checking, is it common practice to have documentation for a set of incoming functions in the source file? I don't think it's entirely necessary since you have to have the name of the class before the function definition

Reply:
Yeah not entirely necessary - I did it hear because there are now function definitions for two separate classes in this single file. I figure it doesn't hurt but I dont feel strongly

Reply:
I think you only added these in the Gemini file so they can probably be deleted, the separation is clear.

Reply:
done